service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceRegisterTask.java

```diff
@@ -307,7 +307,10 @@ private void checkRemainingSchema(Map<String, GetSchemaResponse> scSchemaMap) {
 
       // Currently nothing to do but print a warning
       LOGGER.warn("There are schemas only existing in service center: {}, which means there are interfaces changed. "
-          + "It's recommended to increment microservice version before deploying.", scSchemaMap.keySet());
+              + "It's recommended to increment microservice version before deploying.",
+          scSchemaMap.keySet());
+      LOGGER.warn("ATTENTION: The schemas in new version are less than the old version, "
```

Comment:
I think this warning message is the same as above one. Anyway, it's fine to keep it.

Reply:
This is for the review opinion in another [PR-757](https://github.com/apache/incubator-servicecomb-java-chassis/pull/757). I need to warn users to take care of compatibility issues.
I don't know how to describe it clearer with my poor English  :P